### DISCUSSION TITLE
"Flight Number in Use" message

### DISCUSSION THREAD

**Discussion by zzyzxuk** [flybywiresim/aircraft#2738] [Help] [answered]
I always get this message when starting up the MCDU...  any chance of getting rid of it?  Can't see how helpful it is?

![image](https://user-images.githubusercontent.com/62340504/103157313-7309cf80-47a9-11eb-8f09-843da6d337bd.png)

**Comment by Inaxair** [comment]
Hello

Can you provide more detailed description? What was the step you have done to do it? Any uses of integration like Simbrief?

Thanks
Inaxair from FlyByWire Simulations

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
You start of with flight number "320", if there is someone using the FBW a32nx with the flight number "320" too, that means you have to change it to another flight number

**Comment by zzyzxuk** [comment]
Hello…

 

Yes, here’s more details:

 

I always use Simbrief for planning the flight and I always use this flight number in Simbrief:  BAN1

 

Then when I go into the MCDU I get the error about the ‘flight number in use’.

**Comment by zzyzxuk** [comment]
That seems odd to me.  Why does my PC care what another user on another PC is doing with their flightsim?  Is there some central server linking all of the FBW users together, and if so, isn’t that a bit of an issue GDPR-wise, as I’m not sure I was quite aware that what I was doing is being shared/connected up with other users?

**Reply by zzyzxuk** [reply]
Does this have something to do with the ability to message other users on the MCDU?

**Reply by hotshotp** [reply]
yes - you are flying online with other flybywire people technically if you have telex/free text enabled.

**Reply by MapleNAV-TCASFAULT** [reply]
So to clear it up, someone is flying the a32nx with free text enabled which connects to the FBW servers with the flight number “320” or a flight number same to yours when you spawn in the sim. As a redundant measure to make sure someone doesn’t text two people or cause an error it will tell you “flight number in use” simply change it and everything will be ok. If you don’t want it to pop up, turn off telex in the “options” page of the “mcdu menu”
